Pod/Classes/SeafRepos.m

```diff
@@ -53,13 +53,31 @@ - (id)initWithConnection:(SeafConnection *)aConnection
     return self;
 }
 
-- (BOOL)passwordRequired
-{
-    //Debug("repoId;%@ %d %@", self.repoId, self.encrypted, [connection getRepoPassword:self.repoId]);
-    if (self.encrypted && ![connection getRepoPassword:self.repoId])
-        return YES;
-    else
+- (BOOL)passwordRequired {
```

Comment:
Looks like you changed the logic of this function.  If local decryption is disabled, this function will try to refresh repo password every time it is called and block until refresh is done. This will slow down a lot. And it impacts a lot of existing modules, e.g. SeafFileProvider, etc.

Reply:
现在希望是每次进入加密资料库的时候都向服务器发送密码，有更好的改进方法吗

Reply:
1. 可以考虑加一个函数passwordRequiredWithSyncRefresh来实现新的这个逻辑，然后看看之前调用函数passwordRequired的地方哪些需要改成新的逻辑。
2. 可以记录上一次call self setRepoPassword:password block:^(SeafBase *entry, int ret) 的时间，然后在一定时间内就不要重新调用了